DevoxxClientMobile/src/main/java/com/devoxx/views/SessionsPresenter.java

```diff
@@ -338,6 +339,13 @@ private void updateSessionsDecoration(List<Session> sessions) {
                 colorFlag = !colorFlag;
             }
             session.setDecorated(colorFlag);
+
+            if (previousSessionWithType == null ||
+                    session.getStartDate().toLocalDate().isAfter(previousSessionWithType.getStartDate().toLocalDate()) ||
+                    !session.getTalk().getTalkType().equals(previousSessionWithType.getTalk().getTalkType())) {
+                previousSessionWithType = session;
```

Comment:
When the filter is applied, the same type of talks show the session type above them. For instance if you select only `Conference` type. Shouldn't all these sessions only show it once?

Reply:
I had asked a similar question in [the issue](https://github.com/devoxx/MyDevoxxGluon/issues/68#issuecomment-400244540). Because, it was left open for discussion, I went ahead of what I felt right - Group sessions according to their type but at the same time leave them arranged on the ListView as per their start time.

Reply:
Do you feel we should add a sorting mechanism where a person can choose if they want to sort via session type or session time?

Reply:
I think we have it covered with the current sorting by date/time first, and with the filter to show only certain session types, and now with these labels to group by session type. I'll keep it "simple" (it is pretty complex so far). Adding more options will be probably confusing for the user.